src/server/frontend_wayland/output_manager.cpp

```diff
@@ -105,6 +105,9 @@ auto mf::OutputInstance::output_config_changed(mg::DisplayConfigurationOutput co
         "Fake manufacturer",
         "Fake model",
         mw::Output::Transform::normal);
+
+    send_done();
+
```

Comment:
This is the wrong spot to send the `done` event; it should be sent after *all* the events relevant to the output change have been sent.

The documentation in output_manager.h for `OutputConfigListener` says that `output_config_changed` should return `true` in if a `done` event should be sent, so this will result in *two* done events being sent.